graylog2-server/src/main/java/org/graylog2/rest/resources/users/UsersResource.java

```diff
@@ -556,11 +557,17 @@ public void changePassword(
     @ApiOperation("Update the account status for a user")
     @AuditEvent(type = AuditEventTypes.USER_UPDATE)
     public Response updateAccountStatus(
-            @ApiParam(name = "userId", value = "The id of the user whose status to change.", required = true) @PathParam("userId") String userId,
+            @ApiParam(name = "userId", value = "The id of the user whose status to change.", required = true)
+            @PathParam("userId") @NotBlank String userId,
             @ApiParam(name = "newStatus", value = "The account status to be set", required = true,
-                    defaultValue = "enabled", allowableValues = "enabled,disabled,deleted")
+                      defaultValue = "enabled", allowableValues = "enabled,disabled,deleted")
             @PathParam("newStatus") @NotBlank String newStatusString) throws ValidationException {
 
+        final User currentUser = requireNonNull(getCurrentUser(), "currentUser cannot be null");
```

Comment:
We have `@Context UserContext userContext` for this now.

Reply:
Hah, you are right! :smile: I will change it. Thank you!